libs/contact/src/effects/contact.effects.spec.ts

```diff
@@ -1,104 +1,113 @@
 import { Observable, of } from 'rxjs';
-import { DaffContactEffects } from './contact.effects';
-import { DaffContactUnion, DaffContactServiceInterface } from '@daffodil/contact';
 import { TestBed } from '@angular/core/testing';
-import { DaffTestingContactService } from 'libs/contact/testing/src/drivers/testing/contact.service';
-import { DaffContactDriver } from '../driver/injection-tokens/contact-driver.token';
 import { provideMockActions } from '@ngrx/effects/testing';
-import { DaffContactSubmit, DaffContactSuccessSubmit, DaffContactFailedSubmit, DaffContactRetry, DaffContactCancel } from '../actions/contact.actions';
 import { hot, cold } from 'jasmine-marbles';
 
-describe('DaffContactEffects', () => {
-  let actions$: Observable<any>;
-  let effects: DaffContactEffects<DaffContactUnion, any>;
-  const mockForum = { firstName: 'John', lastName: 'Doe' };
-  let daffContactDriver: DaffContactServiceInterface<DaffContactUnion, any>;
+import {
+	DaffContactSubmit,
+	DaffContactSuccessSubmit,
+	DaffContactFailedSubmit,
+	DaffContactRetry,
+	DaffContactCancel,
+	DaffContactDriver,
```

Comment:
So is this the idea here to use the same driver by having both modules use the code from `dist`?

Reply:
From `src` but, yes.

Reply:
I thought we had all agreed not to do this. That we would use relative paths instead.

Reply:
nvm looks fine